modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -1987,6 +1987,20 @@ void GDScriptAnalyzer::resolve_function_body(GDScriptParser::FunctionNode *p_fun
 	}
 	p_function->resolved_body = true;
 
+	if (p_function->is_abstract) {
+		// Abstract functions don't have a body.
+		if (p_function->body->statements.size() > 0) {
+			push_error(R"(Abstract function cannot have a body.)", p_function->body);
+		}
+		return;
+	} else if (!p_function->is_static) {
```

Comment:
I don't understand why the `!static` is here. Static functions also need to have a body.